homeassistant/components/velux.py

```diff
@@ -54,8 +54,10 @@ def __init__(self, hass, config):
         password = config[DOMAIN].get(CONF_PASSWORD)
         self.pyvlx = PyVLX(
             host=host,
-            password=password)
+            password=password,
+            log_frames=True)
```

Comment:
What does this option do as it sounds like a debugging option, and why is it wanted here? Otherwise (considering the reports that this is working on real devices) this should be ready to be merged.

Reply:
I removed this parameter!